revscoring/features/wikitext/datasources/parsed.py

```diff
@@ -286,6 +293,9 @@ def _extract_tag_name(tag):
 def _extract_template_name(template):
     return str(template.name)
 
+def _extract_sections(template):
+    return mwparserfromhell.wikicode.Wikicode.get_sections(flat=True).template
```

Comment:
Why does this line end in ".template"?

Reply:
We are calling get_sections on template.

Reply:
Looks to me like you are doing something a bit too complicated.  I think you mean to do something like: 
```
        self.sections = Datasource(
            self._name + ".sections",
            _extract_sections, depends_on=[self.wikicode]
        )
```

and 

```
def _extract_sections(wikicode):
    return wikicode.filter_sections(flat=True)
```